Plugins~/Src/MeshSyncClientBlender/msblenContextIntermediatePathProvider.cpp

```diff
@@ -2,9 +2,24 @@
 #include "msblenUtils.h"
 #include <BLI_listbase.h>
 
-std::string append_id(std::string path, const Object* obj) {
+std::string msblenContextIntermediatePathProvider::append_id(std::string path, const Object* obj) {
     auto data = (ID*)obj->data;
-    path += "_" + std::to_string(data->session_uuid);
+    
+    path += "_" + std::string(data->name);
```

Comment:
is this necessary?

Reply:
Yes because the uuid changes every execution

Reply:
the name will be the same right? why are we appending it?